moc_openstack/manifests/suricata.pp

```diff
@@ -0,0 +1,69 @@
+# This manifest manages Suricata IPS
+
+class moc_openstack::suricata {
+
+  package { 'suricata':
+    ensure => 'installed',
+  }
+  service { 'suricata':
+    ensure  => 'running',
+    enable  => true,
+    require => Package['suricata'],
+  }
+
+  file { '/etc/sysconfig/suricata':
+    notify  => Service['suricata'],
+    mode    => '0600',
+    owner   => 'suricata',
+    group   => 'root',
+    require => Package['suricata'],
+    source => 'puppet:///modules/moc_openstack/suricata',
+  }
+
+  file { '/etc/suricata/suricata.yaml':
+    notify  => Service['suricata'],
+    mode    => '0600',
+    owner   => 'suricata',
+    group   => 'root',
+    require => Package['suricata'],
+    source => 'puppet:///modules/moc_openstack/suricata.yaml',
+  }
+    file { '/etc/scripts':
+    ensure => 'directory',
+    owner => 'root',
+    group => 'root',
+    mode  => '0750',
+  }
+    file { 'surnfq':
+    ensure => 'file',
+    content => template('moc_openstack/surnfq.erb'),
+    path => '/etc/scripts/surnfq',
+    owner => 'root',
+    group => 'root',
+    mode  => '0755',
+  }
+    file { 'surupd':
+    ensure => 'file',
+    content => template('moc_openstack/surupd.erb'),
+    path => '/etc/scripts/surupd',
+    owner => 'root',
+    group => 'root',
+    mode  => '0755',
+  }
+
+    file { '/etc/cron.daily/updaterules':
+    ensure => 'link',
+    target => '/etc/scripts/surupd',
+  }
+
+    file { '/etc/cron.d/surnfq.cron':
+    ensure => present,
+    owner => 'root',
+    group => 'root',
+    mode  => '0640',
+    content => "SHELL=/bin/bash\nPATH=/sbin:/bin:/usr/sbin:/usr/bin\nMAILTO=root\n* * * * * root /etc/scripts/surnfq\n",
+  }
+  exec {'updaterules':
+    command => "/etc/scripts/surupd",
+  }
+}
```

Comment:
Looks good to me.

If you require ordering (like exec updaterules should be called after the other files are created, it can be enforced by following "->" notation as its there in other files in same moc_openstack/manifests directory.

Also, maybe we can keep the layout same, like layout of 'file' and '}'. It just makes it easier to follow the code. Otherwise, it looks pretty much good to me.

Like:-
  file { '/etc/cron.daily/updaterules':
    ensure => 'link',
    target => '/etc/scripts/surupd',
  }

Instead of:
    file { '/etc/cron.daily/updaterules':
    ensure => 'link',
    target => '/etc/scripts/surupd',
  }